modules/game/src/main/BinaryFormat.scala

```diff
@@ -92,7 +92,9 @@ object BinaryFormat {
       config.limit - legacyElapsed
 
     def write(clock: Clock): ByteArray = {
-      Array(writeClockLimit(clock.limitSeconds), clock.incrementSeconds.toByte) ++
+      writeClockConfig(clock.config) ++
+        writeClockConfig(clock.players(White).config) ++
+        writeClockConfig(clock.players(Black).config) ++
         writeSignedInt24(legacyElapsed(clock, White).centis) ++
```

Comment:
Currently This probably breaks old clocks/games.
I would suggest the following:
Adding four bytes at the beginning, 00XX where XX is a version tag. Then we should be able to distinguish
between old clock configs (I assume they start with a striclty positive base time) and the new ones.
@ornicar  Let me know, what you think of this.